asr_automation/asr_automation_pipeline.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "13324272",
+      "metadata": {
+        "id": "13324272"
+      },
+      "source": [
+        "<img src=\"https://github.com/superannotateai/automation-suites/blob/main/docs/sa_logo.png?raw=1\" width=\"250\" align=\"left\">"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "e7587e54",
+      "metadata": {
+        "id": "e7587e54"
+      },
+      "source": [
+        "#  ASR with Whisper\n",
+        "\n",
+        "This tutorial walks you through the ```Automatic Speech Recognition``` automation pipeline within SuperAnnotate platform. \n",
+        "\n",
+        "In this tutorial we will go through the following steps:\n",
+        "\n",
+        "1. Setup the environment and install dependences\n",
+        "2. Run the ```OpenAI Whisper``` model on ```US Election 2020 - Presidential Debates``` dataset\n",
+        "3. Setup a ```Video``` project at SuperAnnotate with the dataset audio files\n",
```

Comment:
Can we add some note to clear the moment why we use Video editor for Audio files ? And that Audio editor is coming soon?